content/quiz_process.md

```diff
@@ -7,19 +7,63 @@
         "2-Team 20-Question", "2-Team Overtime", and "3-Team Overtime".
 
 ### 3-Team Quiz
+- There will be 20 numbered questions in a quiz.
+- In the event the quiz ends in a tie, additional units of three overtime questions each will be used until the tie is broken.  Two-team rules will apply in overtime if only two teams are involved.  During overtime, all the numbered questions will be of a different question type.
+- A toss-up question is the result of an error by one team.  The two remaining teams will have the opportunity to jump, but the team making the error is ineligible for that question.
+    - If question 1 to 15 is erred, the next numbered question will be a toss-up question. 
+    - If question 16 is erred, the toss-up question will be question 16A.  If 16A is erred, then 16B is awarded as a 20-point assigned bonus question to the team who has not erred.  This bonus question goes to the corresponding chair of the team that has not erred. 
+- If toss-up question 2 to 15 is erred, then the next numbered question is awarded as a 20-point assigned bonus question to the team who has not erred.  This bonus question goes to the corresponding chair of the team that has not erred. 
+- If question 16 is a toss-up question and is erred, 16A is awarded as a 20-point assigned bonus question to the team who has not erred.  This bonus question goes to the corresponding chair of the team that has not erred.
+- Error points begin at question 17.
+- The first error committed, beginning at question 17, results in a toss-up question, which is numbered 17A, for the two remaining teams.  If there is an error on 17A, the next question will be 17B and will be a 10-point bonus for the team that has not committed an error.  This process applies for the remainder of the quiz.  Three teams must be enlisted to jump on questions 17, 18, 19, and 20.
 
 ### Tie-Breaker Quiz
+- In the event three teams are involved, a 20-question tie-breaker quiz will be used.
+    - The above rules for a three-team, 20-question quiz applies.
+- In the event two teams are involved, a 15-question tie-breaker quiz will be used with the following guidelines:
+    - If an error occurs on questions 1 through 11, the next numbered question is a 20-point assigned bonus to the team who has not erred.  This bonus goes to the corresponding chair of the team that has not erred.   
+    - If questions 12, 13, 14, or 15 are erred, the bonus question will remain on the same question number and will not be counted in the basic 15 questions. Both teams must jump on questions 13, 14, and 15.
+    - All bonus questions up to and including 12B are worth 20 points.
+    - All bonus questions from 13 to the end of the quiz are worth 10 points.
+    - Error points begin at question 13.
+    - A quiz-out is 3 correct questions.
 
```

Comment:
I left out these two from the current rulebook. They sure don't belong under Types of Quizzes. 

"During preliminary rounds of tournaments, if the meet director decides not to break ties in order to keep the tournament on schedule, this will be permitted.  If two teams tie for first place, they will each get first place team points and the other team will get third place team points.  If two teams tie for second place, both teams will get second place team points."

"Districts have the ability to decide at the district or zone level if they would prefer to use a jumped-on Bonus. The bonus question would then be jumped on by the team who has not erred, including quizzers who have quizzed out. "

Reply:
Definitely. Should go in Meet Logistics document.

Reply:
Agreed.